core/qqbouchot.cpp

```diff
@@ -740,6 +742,23 @@ void QQBouchot::parsingFinished()
 		m_history.append(m_newPostHistory);
 		m_lastId = m_parser->maxId();
 		m_state.hasNewPosts = true;
+
+		//On vérifie pour chaque post s'il est le seul pour la minute
+		if (m_bSettings.isShortNorlogeEnabled()) {
```

Comment:
I think that this should be always computed as I short norloge are activated after post is inserted in history it won't work on older posts